coordinator/src/scheduler.rs

```diff
@@ -222,3 +245,42 @@ fn build_remind_to_close_expired_position_notification_job(
         }
     })
 }
+
+fn build_remind_to_close_liquidated_position_notification_job(
+    schedule: &str,
+    notification_sender: mpsc::Sender<Notification>,
+    pool: Pool<ConnectionManager<PgConnection>>,
+) -> Result<Job, JobSchedulerError> {
+    Job::new_async(schedule, move |_, _| {
+        let notification_sender = notification_sender.clone();
+        let mut conn = pool.get().expect("To be able to get a db connection");
```

Comment:
📣 Looks to me like a risky `expect`.

Reply:
https://github.com/get10101/10101/pull/2371